### ISSUE TITLE
Add git source note to Linux install documentation

### ISSUE THREAD

**Issue by leblowl** [openframeworks/openFrameworks#6768]
Hey there! I am new to the project and thus going through setup/compilation for the first time and documenting where I am having trouble. I cloned the repository from github instead of downloading it and am following this install guide: https://github.com/openframeworks/openFrameworks/blob/271863cd2d6ef3901048d9b6aafdd24e360cecfc/docs/linux.md

I found 2 things that were tricky:
1. I needed to run `scripts/linux/download_libs.sh`
2. The examples don't work

And being new to the project, I didn't know where to find this information and just assumed the docs were out of date. So I am thinking it might be helpful to add a little note to the Linux install guide, like "NOTE: If cloning the repository from Github, please refer to https://github.com/openframeworks/openFrameworks/blob/master/INSTALL_FROM_GITHUB.md#how-to-compile-openframeworks-from-github for additional setup steps". 

Also it seems like https://github.com/openframeworks/openFrameworks#developers and https://github.com/openframeworks/openFrameworks/blob/master/INSTALL_FROM_GITHUB.md#how-to-compile-openframeworks-from-github overlap in their information, maybe we should also link to https://github.com/openframeworks/openFrameworks/blob/master/INSTALL_FROM_GITHUB.md#how-to-compile-openframeworks-from-github from the README.

What do you think?

**Comment by arturoc** [comment]
Yeah that sounds good, if you can send a PR that would be really useful
